<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarOrderForm</class>
 <widget class="QMainWindow" name="CarOrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" rowminimumheight="0,0">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,6">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
        <item row="0" column="1">
         <widget class="QLineEdit" name="phoneEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Телефон</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QDateEdit" name="endLeaseDatePicker"/>
          </item>
          <item>
           <widget class="QComboBox" name="endLeaseTimePicker"/>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QDateEdit" name="startLeaseDatePicker"/>
          </item>
          <item>
           <widget class="QComboBox" name="startLeaseTimePicker"/>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Время аренды по</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="firstnameEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="patronymicEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="surnameEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Время аренды с</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QTextEdit" name="commentEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Отчество</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="price">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Комментарий</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Итоговая цена</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="sendOrder">
          <property name="text">
           <string>Оставить заявку</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
        <item>
         <widget class="QLabel" name="CarImage">
          <property name="text">
           <string>CarImage</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carName">
          <property name="text">
           <string>CarName</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_5">
          <property name="verticalSpacing">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Класс авто</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="carClass">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Разгон до 100 км/ч, сек</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="carAceleration">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Мощность, лс</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="carPower">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
